From 96a9c3d9e22e5d2fbe35526488d4ced1237d6c02 Mon Sep 17 00:00:00 2001
From: Vedraj Gawas <gawasvedraj@gmail.com>
Date: Wed, 6 May 2026 19:26:40 +0530
Subject: [PATCH] no_dummy_keystore

---
 kernel/manager/apk_sign.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/kernel/manager/apk_sign.c b/kernel/manager/apk_sign.c
index 04db49b..5637116 100644
--- a/kernel/manager/apk_sign.c
+++ b/kernel/manager/apk_sign.c
@@ -365,8 +365,7 @@ bool is_manager_apk(char *path)
 	}
 #endif
 
-	return (check_v2_signature(path, 0x363, "4359c171f32543394cbc23ef908c4bb94cad7c8087002ba164c8230948c21549") // dummy.keystore
-	|| check_v2_signature(path, EXPECTED_SIZE, EXPECTED_HASH)  // kernelsu official
+	return (check_v2_signature(path, EXPECTED_SIZE, EXPECTED_HASH)  // kernelsu official
 	|| check_v2_signature(path, 0x375, "484fcba6e6c43b1fb09700633bf2fb4758f13cb0b2f4457b80d075084b26c588")  // KOWX712/KernelSU
 	|| check_v2_signature(path, 0x3e6, "79e590113c4c4c0c222978e413a5faa801666957b1212a328e46c00c69821bf7")  // rifsxd/KernelSU-Next
 	|| check_v2_signature(path, 0x396, "f415f4ed9435427e1fdf7f1fccd4dbc07b3d6b8751e4dbcec6f19671f427870b")  // rsuntk/KernelSU
-- 
2.54.0

